### PR TITLE
[DataGrid] Add frFR locale

### PR DESCRIPTION
**Pull request by oliviertassinari**
The build should fail as I have translated a key that isn't listed `filterOperatorAfter` (#1075). We can rebase once fixed.

### CODE REVIEW

**Review by DanailH — APPROVED** [review]

File: packages/grid/_modules_/grid/locales/frFR.ts [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+  // Filter panel text
+  filterPanelAddFilter: 'Ajouter un filtre',
+  filterPanelDeleteIconLabel: 'Supprimer',
+  filterPanelOperators: 'Opérateurs',
+  filterPanelOperatorAnd: 'Et',
+  filterPanelOperatorOr: 'Ou',
+  filterPanelColumns: 'Colonnes',
+  filterPanelInputLabel: 'Valeur',
+  filterPanelInputPlaceholder: 'Filrer la valeur',
```
Comment (dtassone) [inline packages/grid/_modules_/grid/locales/frFR.ts:42]: typo

File: packages/grid/_modules_/grid/locales/frFR.ts [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+  filterOperatorAfter: 'postérieur',
+  filterOperatorOnOrBefore: 'égal ou postérieur',
+  filterOperatorBefore: 'antérieur',
+
+  // Column menu text
+  columnMenuLabel: 'Menu',
+  columnMenuShowColumns: 'Afficher les colonnes',
+  columnMenuFilter: 'Filtre',
+  columnMenuHideColumn: 'Cacher',
+  columnMenuUnsort: 'Unsort',
```
Comment (dtassone) [inline packages/grid/_modules_/grid/locales/frFR.ts:61]: ?

File: packages/grid/_modules_/grid/locales/frFR.ts [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+  columnHeaderSortIconLabel: 'Trier',
+
+  // Rows selected footer text
+  footerRowSelected: (count) =>
+    count !== 1
+      ? `${count.toLocaleString()} lignes sélectionnées`
+      : `${count.toLocaleString()} ligne sélectionnée`,
+
+  // Total rows footer text
+  footerTotalRows: 'Lignes totale:',
```
Comment (dtassone) [inline packages/grid/_modules_/grid/locales/frFR.ts:77]: totales

Comment (oliviertassinari) [reply packages/grid/_modules_/grid/locales/frFR.ts:77]:
```suggestion
  footerTotalRows: 'Lignes totales :',
```

File: packages/grid/_modules_/grid/locales/frFR.ts (hunk 2 above)
Comment (oliviertassinari) [reply packages/grid/_modules_/grid/locales/frFR.ts:61]:
```suggestion
  columnMenuUnsort: 'Annuler le tri',
```

File: packages/grid/_modules_/grid/locales/frFR.ts (hunk 1 above)
Comment (oliviertassinari) [reply packages/grid/_modules_/grid/locales/frFR.ts:42]:
```suggestion
  filterPanelInputPlaceholder: 'Filtrer la valeur',
```

File: packages/grid/_modules_/grid/locales/frFR.ts [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+  filterOperatorNot: "n'est pas",
+  filterOperatorOnOrAfter: 'égal ou postérieur',
+  filterOperatorAfter: 'postérieur',
+  filterOperatorOnOrBefore: 'égal ou postérieur',
+  filterOperatorBefore: 'antérieur',
+
+  // Column menu text
+  columnMenuLabel: 'Menu',
+  columnMenuShowColumns: 'Afficher les colonnes',
+  columnMenuFilter: 'Filtre',
```
Comment (oliviertassinari) [inline packages/grid/_modules_/grid/locales/frFR.ts:59]:
```suggestion
  columnMenuFilter: 'Filtrer',
```

**Comment by DanailH** [comment]
Maybe I'm missing something but the build fails because `filterOperatorAfter` doesn't exist.

File: packages/grid/_modules_/grid/locales/frFR.ts [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+  // Columns selector toolbar button text
+  toolbarColumns: 'Colonnes',
+  toolbarColumnsLabel: 'Choisir les colonnes',
+
+  // Filters toolbar button text
+  toolbarFilters: 'Filtres',
+  toolbarFiltersLabel: 'Afficher les filtres',
+  toolbarFiltersTooltipHide: 'Cacher les filtres',
+  toolbarFiltersTooltipShow: 'Afficher les filtres',
+  toolbarFiltersTooltipActive: (count) => `${count} filtre(s) actif(s)`,
```
Comment (dtassone) [inline packages/grid/_modules_/grid/locales/frFR.ts:25]: why not check the count value and put the 's' accordingly?

File: packages/grid/_modules_/grid/locales/frFR.ts [hunk 6]
```diff
@@ -0,0 +1,81 @@
...
+  columnMenuLabel: 'Menu',
+  columnMenuShowColumns: 'Afficher les colonnes',
+  columnMenuFilter: 'Filtrer',
+  columnMenuHideColumn: 'Cacher',
+  columnMenuUnsort: 'Annuler le tri',
+  columnMenuSortAsc: 'Tri ascendant',
+  columnMenuSortDesc: 'Tri descendant',
+
+  // Column header text
+  columnHeaderFiltersTooltipActive: (count) => `${count} filtre(s) actif(s)`,
```
Comment (dtassone) [inline packages/grid/_modules_/grid/locales/frFR.ts:66]: same here

File: packages/grid/_modules_/grid/locales/frFR.ts (hunk 5 above)
Comment (oliviertassinari) [reply packages/grid/_modules_/grid/locales/frFR.ts:25]: > why not

Because we should start doing it first in the enUS locale?

Comment (oliviertassinari) [reply packages/grid/_modules_/grid/locales/frFR.ts:25]: I'm adding it

**Review by dtassone — APPROVED** [review]

**Pull request merged**